LedBlinker/Top/LedBlinkerTopology.cpp

```diff
@@ -82,7 +83,7 @@ void configureTopology() {
     cmdSeq.allocateBuffer(0, mallocator, CMD_SEQ_BUFFER_SIZE);
 
     // Rate group driver needs a divisor list
-    rateGroupDriver.configure(rateGroupDivisors, FW_NUM_ARRAY_ELEMENTS(rateGroupDivisors));
+    rateGroupDriverComp.configure(rateGroupDivisorsSet, FW_NUM_ARRAY_ELEMENTS(rateGroupDivisorsSet.dividers));
```

Comment:
It seems `rateGroupDriver` was renamed by mistake, could you revert back?

Reply:
Sorry for the mistake, I fixed it